accessibility/README.md

```diff
@@ -0,0 +1,490 @@
+* Images can be categorised in different ways and there are some different recommendations for alt text depending on the type of image.
+* The W3C Web Accessibility Initiative has some good resources around Images:
+  * [Images Tutorial](https://www.w3.org/WAI/tutorials/images/) - Outlines the different purposes an image might have and recommendations for adding accessible text
+  * [`alt` Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) - Provides a helpful decision tree on how to handle different situations with images, including when an empty `alt` attribute is appropriate. 
+
+[ꜛ Back to TOC](#table-of-contents)
+
+## Element IDs
+
+* A variety of accessibility code and functionality (such as the `label` element's `for` attribute, `aria-labelledby` and other `aria-` attributes) use the `id` attribute to reference other elements
+  * For example, linking a checkbox input with a label using `id`/`for` allows the user to click on the label to toggle the checkbox
+  * This helps make the input easier to trigger with a mouse, and also makes it possible to style custom inputs (for example custom checkboxes) by applying styles to the `<label>` element
+* When writing components that use the `id` attribute - ensure that the `id` will always be unique
+  * Avoid hard coding an id for an element unless you know it will only be used once
+  * This is particularly important when the component is designed to be re-used, for example a swatch or a line item component
+  * Unique ids can be generated using passed in data, and additionally namespaces if needed
+  * For section type Canvas components - you could also pass the Shopify `section.id` as a prop and use that in your element id
+
+Example of a dynamic id based on unique prop data: 
+```vue
+<template>
+  <div class="quantity-selector">
+    <label
+      :for="`quantity-selector-input-${lineItem.key}`"
+      class="visually-hidden"
+    >
+      Quantity
+    </label>
+
+    <input
+      :id="`quantity-selector-input-${lineItem.key}`"
+      type="number"
+      value="1" 
+    >
+  </div>
+</template>
+<script>
+export default {
+  name: 'QuantitySelector',
+  
+  props: {
+
+    // Cart line item data from Shopify
+    lineItem: {
+      type: Object,
+    }
+  }
+}
+</script>
+```
+
+Example of setting an explicit unique id with a `namespace` prop
+```vue
+<template>
+  <div class="variant-dropdown">
+    <label
+      :for="`${namespace}-variant-dropdown`"
+      class="visually-hidden"
+    >
+      Quantity
+    </label>
+
+    <select :id="`${namespace}-variant-dropdown`">
+      <option v-for="option in options">
+    </select>
+  </div>
+</template>
+<script>
+export default {
+  name: 'VariantDropdown',
+  
+  props: {
+    namespace: {
+      type: String,
+      default: 'default'
+      required: true,
+    },
+    
+    options: {
+      type: Array,
+      default: () => [],
+
```

Comment:
How nitpicky should we be with code examples 👀 
Looks like there's an extra newline here that can be removed

Reply:
Good spot - Its a little harder to pick up code example styleguide issues when I'm working just in markdown.

Fixed this 👍